api/accounts/accounts-router.js

```diff
@@ -1,27 +1,62 @@
+const Accounts = require(`./accounts-model`)
 const router = require('express').Router()
+const { checkAccountId, checkAccountNameUnique, checkAccountPayload } = require(`./accounts-middleware`)
 
-router.get('/', (req, res, next) => {
-  // DO YOUR MAGIC
+router.get('/', async (req, res, next) => {
+  try {
+    const accounts = await Accounts.getAll()
+    res.json(accounts)
+  } catch (err) {
+    next(err)
+  }
 })
 
-router.get('/:id', (req, res, next) => {
-  // DO YOUR MAGIC
+router.get('/:id', checkAccountId, async (req, res, next) => {
+  try {
+    const account = await Accounts.getById(req.params.id)
```

Comment:
Nit: Since you're already getting the account in your middleware you could put the account within the req in the middleware and then wouldn't have to call to the db twice